### ISSUE TITLE
Fix support for Webpack

### ISSUE THREAD

**Issue by preeteshjain** [caroso1222/notyf#5]
I am using Notyf with Webpack. Here's how I installed Notyf:

```
npm install notyf --save
```

Then I used this code in my [VueJS](http://vuejs.org) component to initialize (`require`) Notyf:

```
var Notyf = require('notyf')
var notyf = new Notyf()
notyf.alert('You must fill out the form before moving forward')
```

I got the following error:

```
Uncaught ReferenceError: Notyf is not defined
```

I have also tried:

```
import Notyf from 'notyf'
```

Which didn't work either.

Can anyone help me getting this to work with VueJS and Webpack? I have used other libraries this way too, and they work just fine. This one isn't working. Any help would be appreciated.

**Comment by caroso1222** [comment]
I've never worked with Vue but I'm going to take a look at it.

**Comment by preeteshjain** [comment]
@caroso1222 Okay. Let me know if you get it to work. I want to know where I am going wrong.

**Comment by caroso1222** [comment]
@preeteshjain 

**TL;DR**
It's fixed. Now you can import Notyf via `var Notyf = required('notyf')`

**Some thoughts**
The problem was you weren't specifying which particular file you were trying to require the module from. I tried `var Notyf = require('notyf/src/notyf')` and worked like a charm. I know this not as clean as we would like to import our modules, so I added the module export at the entry point. I created a Vue project using vue-cli with webpack as the template and successfully imported Notyf this way:

```javascript
var Notyf = require('notyf')
import 'notyf/dist/notyf.min.css'
var notyf = new Notyf()
notyf.alert('Please fill out the form')
```

<img width="1273" alt="screen shot 2017-02-19 at 12 25 35 pm" src="https://cloud.githubusercontent.com/assets/3689856/23104631/9ffca8d2-f69f-11e6-9638-29d5ce372bbf.png">

**Comment by preeteshjain** [comment]
Yes, I too think `var Notyf = require('notyf/src/notyf')` is not a good approach.

Anyway, I'm glad that now it works as expected.
